RELEASING.md

```diff
@@ -49,37 +49,33 @@ You should have no local changes and tests should be passing.
 
 ## Updates for new Calico CRDs
 
-If the release includes new Calico CRDs, add the new CRDs to `hack/gen-csv/get-manifests.sh` and `hack/gen-csv/clusterserviceversion.template`.
+If the release includes new Calico CRDs, add the new CRDs to `hack/gen-bundle/get-manifests.sh` and `config/manifests/bases/operator.clusterserviceversion.yaml`.
 
 ## Publishing a release on RH Catalog
 
 We currently only publish operator releases targeting Calico. If the release targets Calico, continue onto the following steps to generate the
-ClusterServiceVersion for it, and publish the release on the RH Catalog.
+operator bundle for it, and publish the release on the RH Catalog.
 
-1. After the semaphore job in the releasing steps is complete, and images have been tagged and pushed, checkout the tag you released.
+1. After the semaphore job in the releasing steps is complete, and images have been tagged and pushed, checkout the release branch containing the released tag.
```

Comment:
Why this change? Shouldn't we build the bundle from the precise tag?

Reply:
Hmm yeah this is odd, I reverted this phrasing. In practice the release branch head should be at that tag